static/app/views/releases/detail/overview/commitAuthorBreakdown.tsx

```diff
@@ -85,27 +86,28 @@ class CommitAuthorBreakdown extends AsyncComponent<Props, State> {
 
     return (
       <Wrapper>
```

Comment:
Do we still need these `Wrapper`s when we are using `SidebarSection`s?

Reply:
switched back to use `<SectionHeading>` to address component updates in another PR